app/specs/controllers/StudentsShowCtrl.spec.js

```diff
@@ -0,0 +1,48 @@
+// disable max-nested-callbacks linter for this test
+// it is good to have a good separation of contexts inside tests
+
+/* eslint-disable max-nested-callbacks */
+
+'use strict';
+
+define(['paw',
+'angular-mocks',
+'controllers/StudentsShowCtrl'],
+function() {
+  describe('Students Show Ctrl', function() {
+    beforeEach(module('paw'));
+
+    var $controller, controller, expectedStudent;
+
+    beforeEach(inject(
+      function(_$controller_) {
+        $controller = _$controller_;
+        controller = $controller('StudentsShowCtrl');
+        expectedStudent =
+        {
```

Comment:
Unify style: either you put this `{` beside the  above `=` or you fix the `address: {` line